app/assets/javascripts/application.js

```diff
@@ -16,9 +16,4 @@
 //= require popper
 //= require bootstrap
 //= require requests/index
-//= require static_pages.js
-
-require("@rails/ujs").start()
```

Comment:
For rails 5, you don't need to do this i believe, the `require` method gave an error in the frontend JS console.